src/types/index.ts

```diff
@@ -0,0 +1,12 @@
+import { CardButton, ImageProps } from "../components"
+
+export interface SectionInfo {
+  title: string
+  description: string
+  preTitle?: string
+  highlightedWord?: string
+  buttons?: CardButton[]
+  icon?: ImageProps
+  image?: ImageProps
+  rowReversedw: boolean
```

Comment:
```suggestion
  rowReverse: boolean
```

Reply:
Addressed in https://github.com/threshold-network/website/pull/105/commits/f6273f71a2ebc922af66ed7c6d36c28694e5a4e9